.eslintrc.js

```diff
@@ -9,7 +11,8 @@ module.exports = {
     ],
     globals: {
         Atomics: "readonly",
-        SharedArrayBuffer: "readonly"
+        SharedArrayBuffer: "readonly",
+        assert: "readonly"
```

Comment:
why ?

Reply:
free-variable `assert` is used throughout the test-code.  i can remove the global, and add pragma `/* global assert */` to every test-file if that's what you want.

Reply:
Both solutions look like hacks... We can't have `assert` accepted as a global in `api.js`, because it's not defined there, and we shouldn't have to add it as a global everywhere in the tests either, since it's not an external global but a built-in in node. I think the best solution is to have a separate `.eslintrc` for the tests, that inherits from the main `.eslintrc`.

Reply:
fixed in commit 893de98a:
    - remove from /test/ invalid-code containing free-variable assert

after removing invalid-code sections, i verified failed-test behavior is still exactly the same as in `master` branch (when i intentionally failed a test-case).